src/one_dragon/base/push/push_config.py

```
@@ -33,7 +45,38 @@ def __init__(self):
+                migration_map = {
+                    'qywx_app_corp_id': parts[0],
+                    'qywx_app_corp_secret': parts[1],
+                    'qywx_app_to_user': parts[2],
+                    'qywx_app_agent_id': parts[3]
+                }
+                # 可选的media id
+                if len(parts) >= 5:
+                    migration_map['qywx_app_media_id'] = parts[4]
+
+                for new_key, new_value in migration_map.items():
+                    # 只有当新key不存在或为空时，才进行迁移，避免覆盖用户的新设置
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)